ignite/handlers/param_scheduler.py

```diff
@@ -1393,6 +1393,99 @@ def simulate_values(cls, num_events: int, schedulers: List[_LRScheduler], **kwar
             return values
 
 
+class ReduceLROnPlateauScheduler(ParamScheduler):
```

Comment:
We have to include this class to the docs: https://github.com/pytorch/ignite/blob/master/docs/source/handlers.rst#parameter-scheduler